pom.xml

```diff
@@ -27,7 +27,7 @@
   <properties>
     <revision>1</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.319.1</jenkins.version>
+    <jenkins.version>2.346.2</jenkins.version>
```

Comment:
Note that 2.346.1 is recommended: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions